test/system/150-login.bats

```diff
@@ -107,6 +122,65 @@ function setup() {
 # BEGIN actual tests
 # BEGIN primary podman login/push/pull tests
 
+# FIXME: move the two auto-update tests into 255-auto-update.bats once there
+# are helper functions for starting and stopping a registry.
+
+@test "podman-auto-update --authfile"  {
```

Comment:
LGTM at first pass, but please remember to move this to the bottom. This test cannot precede the `basic` one.

Will give another more careful look starting soon.